ckanext/data_qld_theme/templates/scheming/package/resource_read.html

```diff
@@ -29,6 +29,10 @@
     {%- do exclude_fields.append('validation_options') -%}
   {%- endif -%}
 
+  {%- if not h.data_qld_show_resource_visibility(res, c.userobj) -%}
```

Comment:
Use `g.userobj` instead of `c.userobj` for CKAN 2.9 compatibility.
`c` will be deprecated in later CKAN versions

Reply:
refactored

Reply:
> Use `g.userobj` instead of `c.userobj` for CKAN 2.9 compatibility.
> `c` will be deprecated in later CKAN versions

please update pr to include fixes for cicd, we have now got github actions running 2.8 and 2.9 builds so that we know that we are all a ok for our near future upgrade.